### ISSUE TITLE
Support FireMonkey's UserCSS

### ISSUE THREAD

**Issue by erosman** [greasyfork-org/greasyfork#1358]
## Introduction

Personally, I think FireMonkey's userCSS format is simple (especially for new style developers) and more efficient for users, and style developers, as well as style managers.

The deprecated `@-moz-document` or `@document` method of putting multiple styles into one file was original meant for `userChrome.css` which was a single file.

As you are aware from developing Stylish, style managers have to break the segments and apply them individually anyway.

Unfortunately, due to the popularity of the userStyle format and the large volume of existing userStyles, the format has been kept, even though an alternative is available.

## Proposal

Start supporting the userCSS format

## Support

UserCSS format for metadata block is the same as the userScript.

To start with, if there are any userCSS on GreasyFork, they can be marked as FireMonkey only compatible (for now). 
Since #1353, GreasyFork can also verify if FireMonkey (v2.74+) is present.

If desired, #1356 can also convert to userCSS, if there is only one segment (many are like that) and there is no `regexp()` rule.
For those styles, an additional button can be added to show their availability to install as userCSS.

I think userCSS support would encourage the use of the format.

See also:
- #656
- [Support include/exclude in usercss header](https://github.com/openstyles/stylus/issues/343)

**Comment by erosman** [comment]
PS. I have written a UserStyle to UserCSS converter (where possible) which will be in the next update of FireMonkey (v2.74).

**Comment by JasonBarnabe** [comment]
If/when this is supported by other style managers, then I will consider adding support on Greasy Fork. It would also be preferable for there to be a specification/documentation of some sort that could be followed. But right now I see it as just a proprietary alternative to an existing, widespread format.

I don't want to cause confusion with non-developers with different formats, so likely the support would look like:

1. Wait until the majority of users can install this format.
2. Add support for authors to write in either format.
3. Do not convert between formats, just assume the style manager can handle it all (or potentially do a version check to stop it)
4. Change the JS to CSS converter to convert to the new format, if it becomes the most widespread/recommended way.

So in short, I'm not saying "no", but "not until Stylus supports it".
